### DISCUSSION TITLE
[@types/oracledb] missing type definition for ExecuteOptions interface

### DISCUSSION THREAD

**Discussion by icarrascobx** [DefinitelyTyped/DefinitelyTyped#68556] [Issues with a @types package]
ExecuteOptions interface is missing fetchTypeHandler definition.

documentation: [node-oracldb](https://node-oracledb.readthedocs.io/en/latest/api_manual/connection.html#executeoptionsparams)

[a comment by an automated account is not shown]

**Comment by dannyb648** [comment]
Good catch, will add it to the 6.3.0 types coming this weekend.
